python/orca/src/bigdl/orca/data/ray_xshards.py

```diff
@@ -160,6 +159,7 @@ def __init__(self, uuid: str, id_ip_store_rdd: "RDD[Any]",
                  partition_stores: Dict[str, "ActorHandle"]) -> None:
         self.uuid = uuid
         self.rdd = id_ip_store_rdd
+        self.rdd.cache()
```

Comment:
This rdd is only contains partition id, node ip and store name, which is always small. Put it in cache won't affect the memory.
Cache here is only needed for prediction to get the corresponding rdd for prediction results.

When this RayXShards is out-of-scope, this rdd will be uncached.